simapp/app_v2.go

```diff
@@ -333,17 +337,21 @@ func NewSimApp(
 // Name returns the name of the App
 func (app *SimApp) Name() string { return app.BaseApp.Name() }
 
-// InitChainer application update at chain initialization
+// InitChainer initializes the chain with the provided request at the chain's initialization.
```

Comment:
Usually, I am not against more comments. But this simply describe the code, and does not have, imo, any added value.